src/ftrackWidget.ts

```diff
@@ -280,16 +280,8 @@ export interface InitializeOptions {
 
 /** Get current ftrack style */
 export function getStyle() {
```

Comment:
I'd prefer this to be renamed something a bit clearer, and with theme rather than style. `getActiveTheme` maybe?

Reply:
agreed.
Changed now.